package.json

```diff
@@ -29,6 +28,13 @@
   "devDependencies": {
```

Comment:
web vitals should not be in dependancies.
we should update react to latest minor version

Reply:
its good to just copy over all the deps from eth-hooks package.json

Reply:
@ShravanSunder I have copied over most of the relevant dependancies from eth-hooks package.json.
I omitted the `@ethersproject/*` and `hardhat` packages as they were fairly specific to eth-hooks.

Since I was bringing in most of the dependancies, I went ahead and also brought in the other configuration files as well. This PR now supports testing and formatting too.